src/veh_type.cpp

```diff
@@ -171,6 +174,42 @@ void vpart_info::load( JsonObject &jo )
     assign( jo, "bonus", def.bonus );
     assign( jo, "flags", def.flags );
 
+    auto reqs = jo.get_object( "requirements" );
+    if( reqs.has_object( "install" ) ) {
+        auto ins = reqs.get_object( "install" );
+
+        auto sk = ins.get_array( "skills" );
+        if( !sk.empty() ) {
+            def.install_skills.clear();
+        }
+        while( sk.has_more() ) {
+            auto cur = sk.next_array();
+            def.install_skills.emplace( skill_id( cur.get_string( 0 ) ) , cur.size() >= 2 ? cur.get_int( 1 ) : 1 );
+        }
+
+        assign( ins, "time", def.install_moves );
+
+        def.install_reqs.load( ins );
+        def.legacy = false;
+    }
+    if( reqs.has_object( "removal" ) ) {
+        auto rem = reqs.get_object( "removal" );
+
+        auto sk = rem.get_array( "skills" );
+        if( !sk.empty() ) {
+            def.removal_skills.clear();
+        }
+        while( sk.has_more() ) {
+            auto cur = sk.next_array();
+            def.removal_skills.emplace( skill_id( cur.get_string( 0 ) ) , cur.size() >= 2 ? cur.get_int( 1 ) : 1 );
+        }
+
+        assign( rem, "time", def.removal_moves );
+
+        def.removal_reqs.load( rem );
+        def.legacy = false;
+    }
```

Comment:
Same answer as to @BevapDin - once repair requirements are also pushed to JSON we can consolidate the common parts.

There are multiple ways of doing this - one way would be to push the skills data within `requirements` but that also touches crafting code so I want to go with this in a separate PR